examples/model_compress/knowledge_distill/KnowledgeDistill.md

```diff
@@ -0,0 +1,31 @@
+Knowledge Distillation on NNI Compressor
+===
+
+## KnowledgeDistill
+
+Knowledge distillation support while fine-tuning the compressed model, in [Distilling the Knowledge in a Neural Network](https://arxiv.org/abs/1503.02531),  the compressed model is trained to mimic a pre-trained, larger model.  This training setting is also referred to as "teacher-student",  where the large model is the teacher and the small model is the student.
```

Comment:
`Knowledge distillation support while fine-tuning the compressed model`, don't understand...

Reply:
resolved.